backend/app/models/licenses.py

```diff
@@ -13,20 +14,18 @@ class LicenseBase(BaseModel):
     text: str
     url: str
     version: str
-    holders: List[UserOut]
-    expiration_date: datetime = Field(default_factory=datetime.utcnow)
+    holders: List[UserOut] = []
+    expiration_date: datetime = Field(
+        default_factory=datetime.utcnow
+    )  # TODO: shoudn't the default be never?
     allow_download: bool = False
 
 
 class LicenseIn(LicenseBase):
     pass
 
 
-class LicenseDB(Document, LicenseBase):
-    creator: UserOut
-    created: datetime = Field(default_factory=datetime.utcnow)
-    modified: datetime = Field(default_factory=datetime.utcnow)
-
+class LicenseDB(Document, LicenseBase, Provenance):
```

Comment:
I understand that Provenance gets the metadata, but was there error in the older version?